tests/integration/tier1/handle-special-pkgs-case/test_handle_shim_x64_pkg.py

```diff
@@ -1,34 +0,0 @@
-from envparse import env
-
-
-# TODO(r0x0d): Remove this integration test as it probably is not needed anymore.
-def test_handle_shim_x64_pkg(shell, convert2rhel):
-    """Ensure c2r handle the shim-x64 package.
-
-    This test needs to pass on both BIOS and UEFI systems as this packages needs
-    to have it's protection config removed during the conversion.
-    """
-
-    shim_x64_pkg = "shim-x64"
-    # Install the shim-x64 package
-    assert (
-        shell(
-            f"yum install -y {shim_x64_pkg}",
-        ).returncode
-        == 0
-    )
-
-    # run utility until the reboot
-    with convert2rhel(
-        ("-y --no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
-            env.str("RHSM_SERVER_URL"),
-            env.str("RHSM_USERNAME"),
-            env.str("RHSM_PASSWORD"),
-            env.str("RHSM_POOL"),
-        )
-    ) as c2r:
-        pass
-    assert c2r.exitstatus == 0
-
-    # Check that the package is still present on the system
-    assert shell(f"rpm -qi {shim_x64_pkg}").returncode != 0
```

Comment:
We should probably keep this test but modify it to work

Reply:
How exactly we should develop the test for it? Maybe checking if convert2rhel got rid of the package during the conversion? If so, then we already have an test for it. 

We could make sure that the package is not present after the conversion finishes?

Reply:
Gotcha. Is that an integration test then?

Reply:
Yeah, there is an integration test that does that. https://github.com/oamg/convert2rhel/pull/668/files#diff-effa0515e248da3fe738adfe9c21f2977701c9e1c8701bcc86c36acb32415a43R97 You just pass the package name that needs to be installed in the tests, and it will check for package is not present after the conversion

Reply:
Okay. LGTM